test/integration/event-handler-test.ts

```diff
@@ -91,12 +91,12 @@ describe("when a handler throws an error", () => {
     } catch (error: any) {
       expect(error.message).toMatch(/oops/);
 
-      const errors = Array.from(error);
+      const errors = Array.from(error.errors);
```

Comment:
The `AggregateError` class doesn't extend `Iterable<T>` anymore, you have to use the `.errors` property